### PR TITLE
Add a ListRuns module for TkMaps

### PR DESCRIPTION
**Pull request by GuillermoFidalgo**
I would appreciate any tips and suggestions on how to make the code more robust or general. It should however be good enough for our use case ( I think ).

### CODE REVIEW

**Comment by mmusich** [comment]
@GuillermoFidalgo @gbenelli is this PR meant to be merged?

File: proxy.cert [hunk 1]
```diff
@@ -0,0 +1,144 @@
+-----BEGIN CERTIFICATE-----
```
Comment (mmusich) [inline proxy.cert]: I am not sure it is good practice to make the certificate public on gitHub.
Please clarify the usage of this file.

Comment (GuillermoFidalgo) [reply proxy.cert]: Thanks for noticing. 

This is a file that was left over from an earlier version of the code. It is not necessary nor do I think will work anymore if used. I will take it out and push the changes.
The certificate needed for some of the functions to work should be placed in one's own `private/` directory

**Comment by mmusich** [comment]
@GuillermoFidalgo I think also the `__pycache__/ListRuns.cpython-36.pyc` file can be removed from the PR right?

**Pull request closed without merging**